File: 0014-longest-common-prefix/0014-longest-common-prefix.py
class Solution(object):
    def longestCommonPrefix(self, strs):
        """
        :type strs: List[str]
        :rtype: str
        """
        strs.sort()
        str1 = strs[0]
        str2 = strs[len(strs) - 1]
        perfix = ""
        for i in range(min(len(str1), len(str2))):
            if str1[i] == str2[i]:
                perfix += str1[i]
            else:
                break
        return perfix
        

File: 0014-longest-common-prefix/0014-longest-common-prefix.js
/**
 * @param {string[]} strs
 * @return {string}
 */
var longestCommonPrefix = function(strs) {
    strs.sort()
    let str1 = strs[0]
    let str2 = strs[strs.length-1]
    let prefix = ""
    for(let i = 0; i < str1.length && str2.length; i++){
        if(str1[i] === str2[i]){
            prefix += str1[i]
        }
        else{
            break
        }  
    }
    return prefix;
};